idxmgmt/ilm/config.go

```diff
@@ -18,34 +18,117 @@
 package ilm
 
 import (
-	"github.com/elastic/beats/libbeat/common/fmtstr"
+	"github.com/pkg/errors"
+
+	"github.com/elastic/beats/libbeat/common"
 	libilm "github.com/elastic/beats/libbeat/idxmgmt/ilm"
 )
 
-const pattern = "000001"
-
-// Config for ILM supporter
+//Config holds information about ILM mode, overwriting and policies
 type Config struct {
-	Mode       libilm.Mode               `config:"enabled"`
-	PolicyName *fmtstr.EventFormatString `config:"policy_name"`
-	AliasName  *fmtstr.EventFormatString `config:"alias_name"`
-	Event      string                    `config:"event"`
+	Mode          libilm.Mode `config:"enabled"`
+	Overwrite     bool        `config:"overwrite"`
+	RequirePolicy bool        `config:"require_policy"`
+	Policies      []Policy
 }
 
-// Enabled indicates whether or not ILM should be enabled
+//Policy binds together an ILM policy's name and body with an event type
+type Policy struct {
```

Comment:
can we name this `EventPolicy` or something like that to distinguish from the inner `Policy` attribute?